packages/react-native-layout/src/components/flex/Flex.tsx

```diff
@@ -1,61 +1,19 @@
 import { View } from 'react-native'
 
-import type { FlexAlignProps, FlexJustifyProps, FlexProps } from './Flex.props'
+import type { FlexProps } from './Flex.props'
 
-const Flex = ({ testID, children, style, flexStyle }: FlexProps) => {
+const Flex = ({ testID, children, style, ...flexSpecificProps }: FlexProps) => {
   return (
-    <View style={[style, flexStyle]} testID={testID}>
+    <View style={[style, flexSpecificProps]} testID={testID}>
       {children}
     </View>
   )
 }
 
 const FlexColumn = (props: FlexProps) => (
```

Comment:
The props of FlexColumn and FlexRow could be refined slightly:
```suggestion
const FlexColumn = (props: Omit<FlexProps, 'flexDirection'>) => (
```